packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/FlatDataInstanceSetImplementationState.ts

```diff
@@ -255,15 +259,34 @@ export abstract class FlatDataInstanceSetImplementationState extends InstanceSet
       this.mappingElement,
       PropertyExplicitReference.create(property),
       rootInstanceSetImplementation,
-      this.mappingElement,
+      SetImplementationImplicitReference.create(
```

Comment:
Hm, I might need to re-think about this, after talking to @MauricioUyaguari I realized that this just doesn't make a lot of sense. We must not create `ImplicitReference` anywhere in the editor code, that should be done in graph builder code only.